spec/components/validators/post_validator_spec.rb

```diff
@@ -5,6 +5,21 @@
   let(:post) { build(:post) }
   let(:validator) { Validators::PostValidator.new({}) }
 
+  context "post body validation" do
+    it "adds an error for empty raw" do
+      post.raw = ""
+      validator.post_body_validator(post)
+      expect(post.errors.count).to eq(2)
```

Comment:
I would assert for the right error messages here instead of just the count otherwise it may result in false positives.